netmiko/mellanox/mellanox_mlnxos_ssh.py

```diff
@@ -42,7 +67,7 @@ def exit_config_mode(self, exit_config="exit", pattern="#"):
         if self.check_config_mode():
             self.write_channel(self.normalize_cmd(exit_config))
             output = self.read_until_pattern(pattern=pattern)
-            if self.check_config_mode():
+            if not self.check_enable_mode():
```

Comment:
That looks incorrect i.e. at that point we should be checking that we are no longer in config_mode (not whether we are in enable_mode). Was there some reason this was required?

I am probably going to strip this out and just call the `exit_config_mode` in the parent classes (i.e. use `super` on it).

Reply:
Yes that probably makes sense, please go ahead change it.